plugins/inputs/kibana/kibana.go

```diff
@@ -187,15 +199,27 @@ func (k *Kibana) gatherKibanaStatus(baseUrl string, acc telegraf.Accumulator) er
 	tags["status"] = kibanaStatus.Status.Overall.State
 
 	fields["status_code"] = mapHealthStatusToCode(kibanaStatus.Status.Overall.State)
-
-	fields["uptime_ms"] = kibanaStatus.Metrics.UptimeInMillis
 	fields["concurrent_connections"] = kibanaStatus.Metrics.ConcurrentConnections
-	fields["heap_max_bytes"] = kibanaStatus.Metrics.Process.Mem.HeapMaxInBytes
-	fields["heap_used_bytes"] = kibanaStatus.Metrics.Process.Mem.HeapUsedInBytes
 	fields["response_time_avg_ms"] = kibanaStatus.Metrics.ResponseTimes.AvgInMillis
 	fields["response_time_max_ms"] = kibanaStatus.Metrics.ResponseTimes.MaxInMillis
 	fields["requests_per_sec"] = float64(kibanaStatus.Metrics.Requests.Total) / float64(kibanaStatus.Metrics.CollectionIntervalInMilles) * 1000
 
+	Version, err := strconv.ParseFloat(strings.Join(strings.Split(kibanaStatus.Version.Number, ".")[:2], "."), 64)
```

Comment:
This line could panic with unexpected version info, in particular when indexing `[:2]`.  Check the length of the split item to ensure it is long enough.  To match Go naming convention use a lowercase variable `version :=`.

Reply:
@danielnelson code is updated to handle any panic situation